ucm2/MediaTek/mtk-rt5650/mtk-rt5650.conf

```diff
@@ -0,0 +1,17 @@
+Comment "ALC5650 sound card on MediaTek SoC"
+Syntax 4
+
+SectionUseCase."HiFi" {
+	File "/MediaTek/mtk-rt5650/HiFi.conf"
+	Comment "Default"
+}
+
+SectionUseCase."HDMI" {
```

Comment:
I think that the HDMI device may be part of the main use case. It's possible to combine the analog input (microphone) with the HDMI output (display) for example. Like we do in HDA-Intel .

Reply:
I haven't tested combining the HDMI output with the analog input... but yeah that makes sense, I'll test that asap.